src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -53,7 +53,10 @@ abstract class SuluKernel extends Kernel
      */
     public function __construct($environment, $debug, $suluContext = self::CONTEXT_ADMIN)
     {
-        $this->name = $suluContext;
+        if (\property_exists($this, 'name')) {
```

Comment:
Has the `name` property be removed upstream? Do we need to set it at all?

Reply:
We don't know if somebody did use the name instead of the sulu context or did something else with it. I would definitely set it if the property exist to the same as before to avoid this any bc break.

Reply:
You didn't really answer my question :see_no_evil: Has the `name` be removed by Symfony from the `Kernel`?

Reply:
In 5.0 its removed.

Reply:
But we should still set it if somebody using 4.4